server/src/models/Book.js

```diff
@@ -0,0 +1,59 @@
+const Model = require("./Model");
+
+class Book extends Model {
+  static get tableName() {
+    return "books";
+  }
+
+  static get jsonSchema() {
+    return {
+      type: "object",
+      required: ["title", "author"],
+      properties: {
+        title: { type: "string", minLength: 1 },
+        Author: { type: "string", minLength: 1 },
+      },
+    };
+  }
+
+  static get relationMappings() {
+    const { BookList, BookFavorites, User } = require("./index.js");
+
+    return {
+      bookLists: {
+        relation: Model.ManyToManyRelation,
+        modelClass: BookList,
+        join: {
+          from: "books.id",
+          through: {
+            from: "bookFavorites.bookId",
+            to: "bookFavorites.bookListId",
+          },
+          to: "bookLists.id",
+        },
+      },
+
+      user: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: User,
+        join: {
+          from: "books.userId",
+          to: "users.id",
+        },
+      },
+      bookFavorites: {
+        relation: Model.ManyToManyRelation,
+        modelClass: BookFavorites,
+        join: {
+          from: "books.id",
+          through: {
+            from: "bookFavorites.bookListId",
+            to: "bookFavorites.bookId",
+          },
+          to: "bookLists.id",
+        },
```

Comment:
a) you might not need this relation

b) if you do need it, it should be a HasMany relation. 
c) you would then need to change lines 47 to 53 accoridngly